### ISSUE TITLE
и снова проблема со структурой

### ISSUE THREAD

**Issue by jvsntsn** [abaevdict/abaevdict-tei#2]
Вот здесь очень интересный кусок, который как бы subentry, но там, где он должен стоять, re не ставится...
Пока сделала ноутом, но мне так не очень нравится

![image](https://user-images.githubusercontent.com/48504970/153236977-6f4302d5-a984-49d8-b55f-0c849061ec24.png)
![image](https://user-images.githubusercontent.com/48504970/153237158-63e7c47b-132e-412b-bc08-76a54c4e14d4.png)

**Comment by solanum-tuberosum** [comment]
да, это постсабэнтри со своими примерами же? если последнее ре поставилось, то и это по идее должно поставиться, нет?

**Comment by jvsntsn** [comment]
Проверю, уже не помню, почему не получилось. Но двоеточие точно не поставится, см. porti
![image](https://user-images.githubusercontent.com/48504970/155679960-7af00a18-0b18-4d79-ab96-2a4b40c16bc9.png)

**Comment by jvsntsn** [comment]
Все, я поняла, в чем прикол. Переделываю и закрываю ишью

**Comment by jvsntsn** [comment]
Но теперь у меня не получается сделать тире перед вторым сабэнтри...
![image](https://user-images.githubusercontent.com/48504970/155683050-af1318fd-09b7-47f1-9639-318ae926eefe.png)
